components/Step1/Step1.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {useEffect, useState} from 'react';
+import {Image, KeyboardTypeOptions, Text, View} from 'react-native';
+import {useWizard} from 'react-use-wizard';
+import {ButtonType, Data} from '../../types/CommonTypes';
+import constants from '../../utils/constants';
+import Form from '../Form/Form';
+import {useDispatch, useSelector} from 'react-redux';
+import {updateUserData} from '../../redux/reducers/WizardData';
+
+import globalStyle from '../../assets/styles/globalStyle';
+
+export const Step1 = () => {
+  const dispatch = useDispatch();
+  const {nextStep} = useWizard();
+  const [formData, setFormData] = useState({});
+  const wizardData = useSelector(
+    (state: {
+      wizard: {name: string; age: number; weight: number; height: number};
+    }) => state.wizard,
+  );
+
+  useEffect(() => {
+    setFormData(wizardData);
+  }, []);
```

Comment:
Why use `formData` state, when you can just read `wizardData` instead?

This seems like an extra and unnecessary `useEffect` + `useState`

Reply:
Instead of passing `formData` [here](https://github.com/YordankaIv/FitnessApp/pull/4/files#diff-20f966714064f13081791a0aab126fceaac1ee23bd4a9d3dd0ae470ee0a95578R89) you can just pass `wizardData`